gossip/emitter/emitter.go

```diff
@@ -169,6 +172,8 @@ func NewEmitter(
 		Periodic:      logger.Periodic{Instance: logger.New()},
 		baseFeeSource: baseFeeSource,
 		errorLock:     errorLock,
+		eventEmissionThrottler: *throttling.NewThrottlingState(
```

Comment:
We might want to have a `nil` option to disable it if desired by the users. Initially we would do an opt-in, then an opt-out, and eventually maybe a forced enabling. Wdyt?